cmd/gardener-extension-provider-kubevirt/app/app.go

```diff
@@ -144,8 +144,6 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 
 			// apply config options
 			configFileOpts.Completed().ApplyETCDStorage(&kubevirtcontrolplaneexposure.DefaultAddOptions.ETCDStorage)
-			configFileOpts.Completed().ApplyGardenId(&kubevirtcontrolplane.DefaultAddOptions.GardenId)
```

Comment:
The garden IDs are currently not used. I believe this code was copied over from vsphere, but the reason to pass and use the garden ID in controllers are vsphere-specific. None of the other provider extensions are using garden IDs. Therefore I decided to remove this code. It could always be reintroduced if really needed.